File: src/main/java/com/testsmirk/GirlProperties.java
package com.testsmirk;

import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

/**
 * Created by testsmirk on 17-1-27.
 */
@Component
@ConfigurationProperties(prefix = "girl")
public class GirlProperties {
    private String cupSize;
    private Integer age;

    public String getCupSize() {
        return cupSize;
    }

    public void setCupSize(String cupSize) {
        this.cupSize = cupSize;
    }

    public Integer getAgel() {
        return age;
    }

    public void setAgel(Integer age) {
        this.age = age;
    }

}

File: src/main/java/com/testsmirk/GirlProperties.kt
package com.testsmirk

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

/**
 * Created by testsmirk on 17-1-27.
 */
@Component
@ConfigurationProperties(prefix = "girl")
class GirlProperties {
    var cupSize: String? = null
    var agel: Int? = null

}
